docs/p2p-connection.md

```diff
@@ -3,7 +3,7 @@
   <p>This section is highly experimental. We will update it soon.</p>
 </div>
 
-The `fetchai/p2p_libp2p:0.2.0` connection allows AEAs to create a peer-to-peer communication network. In particular, the connection creates an overlay network which maps agents' public keys to IP addresses.
```

Comment:
@marcofavorito this has already been bumped and therefore `0.2.0` is enough (you can check by running `aea search connections --query p2p` and you will see highest released version is `0.1.0`

Reply:
thanks, will fix that